tests/components/freebox/test_sensor.py

```diff
@@ -0,0 +1,34 @@
+"""Tests for the Freebox sensors."""
+from copy import deepcopy
+from unittest.mock import Mock
+
+from freezegun.api import FrozenDateTimeFactory
+
+from homeassistant.components.freebox import SCAN_INTERVAL
+from homeassistant.components.sensor import DOMAIN as SENSOR_DOMAIN
+from homeassistant.core import HomeAssistant
+
+from .common import setup_platform
+from .const import DATA_STORAGE_GET_DISKS
+
+from tests.common import async_fire_time_changed
+
+
+async def test_disk(
+    hass: HomeAssistant, freezer: FrozenDateTimeFactory, router: Mock
+) -> None:
+    """Test disk sensor."""
+    await setup_platform(hass, SENSOR_DOMAIN)
+
+    assert hass.states.get("sensor.freebox_free_space").state == "88.27"
```

Comment:
Can we add an assert for total bytes here so we see the math adds up?

Reply:
Addressed